### PR TITLE
Add observability-policies app

### PR DESCRIPTION
**Pull request by marieroque**
towards https://github.com/giantswarm/giantswarm/issues/31155

### Testing

Description on how default-apps-cloud-director can be tested.

- [ ] fresh install works
- [ ] upgrade from previous version works

### Checklist

- [x] Update changelog in CHANGELOG.md, **check all commits are in it before release (renovate included)**.
- [x] Make sure `values.yaml` and `values.schema.json` are valid.

### Trigger e2e tests

/run cluster-test-suites

### CODE REVIEW

File: helm/default-apps-cloud-director/values.yaml [hunk 1]
```diff
@@ -188,6 +188,18 @@ apps:
...
+    appName: observability-policies
+    chartName: observability-policies
+    catalog: default
+    clusterValues:
+      configMap: false
+      secret: false
+    dependsOn: kyverno-crds
+    namespace: kube-system
+    # used by renovate
+    # repo: giantswarm/observability-policies
```
Comment (marieroque) [inline helm/default-apps-cloud-director/values.yaml:201]:
```suggestion
    # repo: giantswarm/observability-policies-app
```

**Review by glitchcrab — APPROVED** [review]

**Comment by glitchcrab** [comment]
also skipped CI here as the tests have passed

**Pull request merged**